Src/Particle/AMReX_ParticleInit.H

```diff
@@ -989,7 +989,17 @@ InitRandom (Long                    icount,
     {
         if(icount*AMREX_SPACEDIM >= std::numeric_limits<int>::max())
         {
-            amrex::Abort("InitRandom has serialize=true, but this would cause too much particle data to be sent from IOProc. Set serialize=false, or use fewer than "+std::to_string(amrex::Math::ceil(std::numeric_limits<int>::max()/amrex::Real(AMREX_SPACEDIM)))+ "particles");
+            amrex::Abort(
+                "InitRandom has serialize=true, but this would cause too much "
+                "particle data to be sent from IOProc. Set serialize=false, "
+                "or use fewer than " +
+                std::to_string(
+                    amrex::Math::ceil(
+                        amrex::Real(std::numeric_limits<int>::max()) /
+                        amrex::Real(AMREX_SPACEDIM)
+                    )
+                ) + "particles"
```

Comment:
Contrary to the original code, we could add a space here.
```suggestion
                ) + " particles."
```